pkg/config/config_test.go

```diff
@@ -117,5 +117,5 @@ func TestServerConfigValidateAndAdjust(t *testing.T) {
 	require.Regexp(t, ".*does not contain a port", conf.ValidateAndAdjust())
 	conf.AdvertiseAddr = "advertise:1234"
 	conf.PerTableMemoryQuota = 1
```

Comment:
Can add a test, when `conf. PerTableMemoryQuota` is zero, `ValidateAndAdjust` will reset it to default quota